README.md

```diff
@@ -138,6 +139,27 @@ You can also override these setting, such as with
 
     WIN81_X64_PRO := file:///Volumes/MSDN/en_windows_8.1_professional_vl_with_update_x64_dvd_4065194.iso
 
+### Packer Global Configuration
```

Comment:
As semi-discussed in #83, the addition of allowing users to customize the configuration during the floppy step (which includes the update to all of the templates) is different from the consolidation of the download methods that you did to implement proxy support. Can you isolate the configuration along with the relevant documentation into its own PR?